common/src/main/java/apoc/util/Util.java

```diff
@@ -168,6 +168,19 @@ public static Label[] labels(Object labelNames) {
         return new Label[] {Label.label(labelNames.toString())};
     }
 
+    public static Label[] getLabelsArray(Node node) {
+        List<Label> labels = getLabels(node);
+        return labels.toArray(new Label[0]);
```

Comment:
You can go directly from stream to array

```suggestion
        return getLabelsAsStream(node).toArray(i -> new Label[i]);
```